plugin/editorconfig.vim

```diff
@@ -192,12 +196,16 @@ function! s:GetFilenames(path, filename) " {{{1
     return l:path_list
 endfunction " }}}1
 
-function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
+function! s:UseConfigFiles(allowEmpty) abort " Apply config to the current buffer {{{1
```

Comment:
Where is the `allowEmpty` parameter used?  I can't find it.  If it is not used, would you please remove it?

Reply:
Indeed, it's a leftover I forgot to remove